### PR TITLE
Add theme-stack to community theme list

### PR DESCRIPTION
**Pull request by jiewenhuang**
添加主题[halo-theme-stack](https://github.com/jiewenhuang/halo-theme-stack) 到主题>社区列表

```release-note
None

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,8 @@
 - [theme-higan](https://github.com/guqing/halo-theme-higan) - 适用于 Halo 2.0 的 Higan 主题，移植于 [hexo-theme-cactus](https://github.com/probberechts/hexo-theme-cactus.git)
 - [theme-sakura](https://github.com/LIlGG/halo-theme-sakura/tree/next) - 适用于 Halo 2.0 的 Sakura 主题
 - [theme-terminal](https://github.com/wan92hen/theme-terminal) - 一款 Terminal 风格的 Halo 2.0 主题
+- [theme-macwen](https://github.com/jiewenhuang/halo-theme-macwen) - 适用于Halo 2.0 的应用下载类主题
```
Comment (wan92hen) [inline README.md:33]:
```suggestion
- [theme-macwen](https://github.com/jiewenhuang/halo-theme-macwen) - 适用于 Halo 2.0 的应用下载类主题
```

File: README.md [hunk 2]
```diff
@@ -30,6 +30,8 @@
...
+- [theme-stack](https://github.com/jiewenhuang/halo-theme-stack) - 移植于[hugo-theme-stack](https://github.com/CaiJimmy/hugo-theme-stack)的个人博客主题
```
Comment (wan92hen) [inline README.md:34]:
```suggestion
- [theme-stack](https://github.com/jiewenhuang/halo-theme-stack) - 移植于 [hugo-theme-stack](https://github.com/CaiJimmy/hugo-theme-stack) 的个人博客主题
```

**Review by wan92hen — CHANGES_REQUESTED** [review]

**Review by ruibaby — COMMENTED** [review]
/lgtm

**Review by JohnNiang — APPROVED** [review]
/lgtm

**Pull request merged**